Move to 1.7.0.Beta0 as preparation for Quarkus 3.20 release processing

michalvavrik opened the pull request:
### Summary

For Quarkus 3.20, dedicated 1.6.z branch has been created, even though CI preparation and some refinement is still work in a progress. We need to move to 1.7.x as Quarkus main is already on 3.21. https://github.com/quarkus-qe/quarkus-test-framework/blob/main/RELEASE.md doesn't say if beta should start from 0 or 1, so I chose zero to match suggested 1.6.0 patch version.

Please check the relevant options

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Dependency update
- [ ] Refactoring
- [x] Release (follows conventions described in the [RELEASE.md](https://github.com/quarkus-qe/quarkus-test-framework/blob/main/RELEASE.md))
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update
- [ ] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [x] Example scenarios has been updated / added
- [x] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

michalvavrik commented:
release workflow failed over 

```
Error: Unable to resolve action `radcortez/project-metadata-action@master`, unable to find version `master`
```
putting this onhold for now

gtroitsk reviewed (changes requested):
Please fix typo in commit message, zero is missing before Beta `1.7.0.Beta0`

michalvavrik commented:
> Please fix typo in commit message, zero is missing before Beta `1.7.0.Beta0`

good catch, fixed; I am going to merge it since I addressed request for change in the commit message and you approved before
